tests/tet/dummy.with_tet_support=off.output

```diff
@@ -0,0 +1,2 @@
+
+DEAL::deal.II is configured without TET support.
```

Comment:
I don't think you need the two variants -- the test is simply never picked up when support is `off`. In other words, if the test for whatever reason is run without support (and consequently outputs `without`), then the test pickup logic is borken, the test fails the comparison to the one available output file, and that is exactly what should happen.